src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1189,7 +1189,7 @@ class AudiusAPIClient {
     return encodedUserId ? decodeHashId(encodedUserId.toString()) : null
   }
 
-  async getUserChallenges({ userID }: GetUserChallengesArgs) {
+  getUserChallenges = async ({ userID }: GetUserChallengesArgs) => {
```

Comment:
Since we use `this` in this function, I need to use arrow functions so that other callers don't override `this`. See: https://www.typescriptlang.org/play#example/this

I could also use `bind()`at the place where I use call but that doesn't let me mock it properly since it creates an entirely new function (and generally looks pretty ugly).

Could maybe sneak around this using `spy` but really curious what people think about using arrow functions here instead.

Reply:
always use arrow imo. it's more normal function semantics and i like consistency